app/uk/gov/hmrc/tradergoodsprofiles/services/RouterService.scala

```diff
@@ -113,4 +118,31 @@ class RouterServiceImpl @Inject() (
           ).toResult
         )
     }
+
+  override def removeRecord(eoriNumber: String, recordId: String, actorId: String)(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, Result, Unit] =
+    EitherT(
+      routerConnector
+        .put(eoriNumber, recordId, actorId)
+        .map {
+          case httpResponse if is2xx(httpResponse.status) => Right(())
+          case httpResponse                               => Left(handleError(httpResponse.body, httpResponse.status, eoriNumber, recordId))
+        }
+        .recover {
+          case UpstreamErrorResponse(message, status, _, _) => Left(handleError(message, status, eoriNumber, recordId))
```

Comment:
Is this reachable code? I dont think this is never reached. I cannot get a test falling into this route. So this case may be deleted.